src/components/Cards.js

```diff
@@ -1,48 +1,26 @@
 import React from 'react'
 import CardItem from './CardItem'
 import './Cards.css';
-
+import Carddata from './Carddata';
 function Cards() {
     return (
         <div className='cards'>
             <h1>Check out these Destination</h1>
             <div className='cards__container'>
                 <div className='cards__wrapper'>
                     <ul className='cards__items'>
```

Comment:
LGTM but we need to implement grid cols so the UI is not disturbed if we add more cards